bucket/folder-marker.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": "4.6",
+    "description": "A tool to label folders with color-coded / image-coded icon",
+    "homepage": "https://foldermarker.com/en/",
+    "license": "Freeware",
+    "url": "https://foldermarker.com/FolderMarker_Free.exe",
+    "hash": "f86645659df6703f90d617c36f0db214a68ce0a240c97a8e4f23219ba4891404",
+    "innosetup": true,
+    "pre_install": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\" }",
+        "",
+        "Rename-Item \"$dir\\icl\\fColors,3.icl\" 'fColors.icl'; Remove-Item \"$dir\\icl\\fColors,*.icl\"",
+        "Rename-Item \"$dir\\icl\\fMain,3.icl\" 'fMain.icl'; Remove-Item \"$dir\\icl\\fMain,*.icl\"",
+        "",
+        "Invoke-ExternalCommand regsvr32 -ArgumentList @(\"`\"$dir\\FMADM.dll`\"\", '/s') -RunAs | Out-Null",
+        "Invoke-ExternalCommand regsvr32 -ArgumentList @(\"`\"$dir\\ShellExt.dll`\"\", '/s') -RunAs | Out-Null",
+        "if ($architecture -eq '64bit') {",
+        "    Invoke-ExternalCommand regsvr32 -ArgumentList @(\"`\"$dir\\ShellExt64.dll`\"\", '/s') -RunAs | Out-Null",
+        "}"
+    ],
+    "pre_uninstall": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\" }",
+        "Invoke-ExternalCommand regsvr32 -ArgumentList @('/u', \"`\"$dir\\FMADM.dll`\"\", '/s') -RunAs | Out-Null",
+        "Invoke-ExternalCommand regsvr32 -ArgumentList @('/u', \"`\"$dir\\ShellExt.dll`\"\", '/s') -RunAs | Out-Null",
+        "if ($architecture -eq '64bit') {",
+        "    Invoke-ExternalCommand regsvr32 -ArgumentList @('/u', \"`\"$dir\\ShellExt64.dll`\"\", '/s') -RunAs | Out-Null",
+        "}",
+        "# Restart explorer so that ShellExt(-64).dll can be removed",
+        "Stop-Process -Name 'explorer'; Start-Sleep -Seconds 2",
+        "if (!(Get-Process -Name 'explorer' -ErrorAction SilentlyContinue)) { Start-Process 'explorer' }"
+    ],
+    "env_set": {
+        "FOLDER_MARKER_DIR": "$dir"
```

Comment:
I'm curious to know why this env var is required?

Reply:
If we don't add this, the app will not load the icons properly.

Reply:
Does the installer also create this?

Reply:
Yep.